config/routes.rb

```diff
@@ -6,4 +6,5 @@
   get '/seances', to: 'seances#index'
   get '/tickets', to: 'tickets#index'
   get '/reservations', to: 'reservations#index'
+  get '/create_hall', to: 'halls#create'
```

Comment:
[This](https://github.com/lukasz-ligizynski/Cinema/pull/12#discussion_r804406703) comment is relevant also here

Reply:
Nvm, I've realized that you fixed it in another PR

Reply:
Yeah, sorry about it. I think I made too many PRs and some changes are in one place, and some in other